### PR TITLE
Forms for self-sourced-project component

### PR DESCRIPTION
**Pull request by TimurGolovinov**
I added some simple forms to self-sourced-project component. There is obviously some room for the further enhancements, like date pickers, form validation etc. I just want to make sure that I am on the right track with what I have done already. I look forward for your feedback. Thanks.

### CODE REVIEW

File: src/app/self-sourced-project/self-sourced-project.component.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 
+
```
Comment (AlexZhidkov) [inline src/app/self-sourced-project/self-sourced-project.component.ts]: When you commit your changes it is a good practice to go over all files being committed and review changes to make sure you don't introduce any unwanted changes. You can easily do this in VS Code (are you using VS Code?) in SOURCE CONTROL by double clicking files - it'll open comparison window. I wouldn't commit changes to this file. You can roll then back during review.

**Comment by AlexZhidkov** [comment]
Thanks for the commit. That's exactly what I wanted. I'll test it later today.

File: src/app/self-sourced-project/self-sourced-project.component.html [hunk 2]
```diff
@@ -1,3 +1,107 @@
-<p>
-  self-sourced-project works!
-</p>
+<h2 class="main-title">Student Placement Arrangement</h2>
```
Comment (AlexZhidkov) [inline src/app/self-sourced-project/self-sourced-project.component.html]: We're using https://material.angular.io/components/categories for our UI components. And we need to make sure that our app looks good on mobile, so it's responsive to screen size. Please have a look at how eoi-business.component.html and eoi-student.component.html are done. Would you be able to do the same? 
To see them in action go to sidenav Student -> Web Site -> Apply. You must be logged in with Google to do so.

File: src/app/self-sourced-project/self-sourced-project.component.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.form-container {
```
Comment (AlexZhidkov) [inline src/app/self-sourced-project/self-sourced-project.component.scss]: We want to use minimum custom css, so it's easier and cheaper to maintain.

File: src/app/self-sourced-project/self-sourced-project.component.ts (hunk 1 above)
Comment (TimurGolovinov) [reply src/app/self-sourced-project/self-sourced-project.component.ts]: Thanks, Alex, I found that feature!

**Pull request merged**